package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@frontify/arcade",
-    "version": "0.16.2",
+    "version": "0.17.0",
```

Comment:
Note that we have breaking changes on the interface. Currently I am not yet familiar on how to handle them between the repositories. (The default for semantic versioning would be to increase the major version)

Reply:
Should be taken care of outside of PR when creating the release